docs/pages/guides/testing-rsc.mdx

```diff
@@ -116,4 +114,4 @@ React Server Components support package exports by default, you can use the `rea
 }
 ```
 
-When a process is bundling for React Server Components, all modules are bundled in React Server-mode and you opt-out with the `"use client"` directive. When `"use client"` is found, the module becomes an async reference to the client module. `"use server"` is not the opposite of `"use client"`, it is instead used to define React Server Actions.
+When a process is bundling for RSC, all modules are bundled in React Server mode and you can opt-out with the `"use client"` directive. When `"use client"` is found, the module becomes an async reference to the client module. `"use server"` is not the opposite of `"use client"`. It is instead used to define React Server Actions.
```

Comment:
"a process" is imprecise. I'd either replace it with a more precise noun or leave it out altogether and say, "When bundling for RSC".

De-hyphenate "opt out", it's not an adjective here.

```suggestion
When bundling for RSC, all modules are bundled in React Server mode and you can opt out with the `"use client"` directive. When `"use client"` is found, the module becomes an async reference to the client module.

`"use server"` is not the opposite of `"use client"`. It is instead used to define React Server Actions.
```